mne/viz/tests/test_topomap.py

```diff
@@ -55,91 +55,100 @@ def test_plot_topomap():
     import matplotlib.pyplot as plt
     from matplotlib.patches import Circle
     # evoked
-    warnings.simplefilter('always', UserWarning)
+    warnings.simplefilter('always')
     res = 16
+    evoked = read_evokeds(evoked_fname, 'Left Auditory',
+                          baseline=(None, 0))
+    ev_bad = evoked.pick_types(meg=False, eeg=True, copy=True)
+    ev_bad.pick_channels(ev_bad.ch_names[:2])
+    ev_bad.plot_topomap()  # auto, should plot EEG
+    assert_raises(ValueError, ev_bad.plot_topomap, ch_type='mag')
```

Comment:
These four lines are the only new ones in this test, the rest were un-indenting a bunch of lines that really shouldn't have been indented by a `with warnings.catch_warnings` statement.